api_compatibility_policy.md

```diff
@@ -43,21 +40,41 @@ alpha:
 * `Pipeline`
 * `PipelineRun`
 
-We are following [the Kubernetes definitions of these stages](https://kubernetes.io/docs/reference/using-api/api-overview/#api-versioning)
-and we are following [the Kubernetes deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/).
-
-* Alpha:
-  * These features may be dropped at any time, though you will be given at least
-    one release worth of warning.
-* Beta:
-  * These features will not be dropped, though the details may change.
-  * Any [backwards incompatible changes](#backwards-incompatible-changes) must be
-    introduced in a backwards compatible manner first, with a deprecation warning
-    in the release notes and migration instructions.
-  * You will be given at least 9 months to migrate before a backward incompatible
-    change is made. This means an older beta API version will continue to be
-    supported in new releases for a period of at least 9 months from the time a
-    newer version is made available.
+### Alpha
+
+- For Alpha CRDs, the `apiVersion` contains the `alpha` (e.g. `v1alpha1`)
+  
+- Alpha feature in beta or stable CRDs are disabled by default and must be enabled by [setting `enable-api-fields` to `alpha`](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#alpha-features)
+
+- These features may be dropped at any time, though you will be given at least one release worth of warning.
+
+### Beta
+
+- The `apiVersion` field of the CRD contains contain `beta` (for example, `v1beta1`).
```

Comment:
nit: we might want one less contain here

Reply:
indeed